_test/data/pages/wiki/syntax.txt

```diff
@@ -459,7 +459,7 @@ echo '</tr></table>';
 | reverse    | display the last items in the feed first |
 | author     | show item authors names |
 | date       | show item dates |
-| description| show the item description. If [[doku>config:htmlok|HTML]] is disabled all tags will be stripped |
```

Comment:
The php/htmlok section bit higher is not yet removed, already fixed in real page.

Reply:
This wiki/syntax, in the unit test folder, deviates from the main one.